src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1393,17 +1394,15 @@ internal static SqlConnection OpenSqlConnection(ConnectionInfo connInfo, string
                 connInfo.ConnectionDetails.Pooling = originalPooling;
 
                 // open a dedicated binding server connection
-                using (SqlConnection sqlConn = new SqlConnection(connectionString))
```

Comment:
Why are we getting rid of the using code block. Was the earlier code closing the connection even though it was being returned?

Reply:
Would this result in a leak?

Reply:
I originally added it because we were Opening the connection. I can add it back.. I'm surprised that it wasn't disposing it once it left this function

Reply:
Let's add the using statement back if there is no reason to remove it.